src/dra/5298C.c

```diff
@@ -784,7 +784,57 @@ void func_800F6BEC(MenuContext* context) {
 
 INCLUDE_ASM("dra/nonmatchings/5298C", func_800F6CC0);
 
-INCLUDE_ASM("dra/nonmatchings/5298C", func_800F6DC8);
+void DrawTimeAttackMenu(MenuContext* ctx) {
+    s16 cursorX;
+    s32 temp_s0;
+    s32 temp_s1;
+    s32 newCursorY;
+    s32 offsetY;
+    s32 i;
+    s32 time;
+
+    i = 0;
+    offsetY = 8;
+    while (i < 12) {
+        cursorX = ctx->cursorX;
+        temp_s0 = i + D_8003C9F0;
+        newCursorY = ctx->cursorY + offsetY;
+        DrawMenuInt(temp_s0 + 1, cursorX + 16, newCursorY, ctx);
+#if defined(VERSION_US)
```

Comment:
I think the blocks of each version are the same except for some constant values so I think you could do something like
```
#if defined(VERSION_US)
#define POS 12345
#else
#define POS 23456
#endif
```
etc.